src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -13,28 +13,28 @@ public class Exercise extends ExerciseBase {
 
 
     // 1. Create an integer member named age with a value of 32
-
+          int age = 32;
 
 
     // 2. Create a String member named firstName with a value of "Jane"
-
+          String member = firstName = "Jane";
```

Comment:
Hi Hassan
 Where it says "Create a String member named firstName", it just means do:
```java
String firstName = "Jane";
```
It doesn't need the `String member = firstName = "Jane";` part you have here.
Cheers
Dave